### PR TITLE
Docs for shared minter suite

### PR DESCRIPTION
**Pull request by ryley-o**
## Description of the change

Update this repo's documentation to reflect the new shared minter suite.

These updates continue to describe the legacy minter suite while we kick off migrations to the new minter suite. Therefore, they may be merged immediately.

---
Asana task: https://app.asana.com/0/0/1205139431852179/f

### CODE REVIEW

File: packages/contracts/README.md [hunk 1]
```diff
@@ -153,17 +160,18 @@ Given that it is an externally linked library with a shared public deployment, t
 - V1 `BytecodeStorageReader` (arbitrum-goerli): https://goerli.arbiscan.io/address/0x681861cD4fC92d70aE57745385065ef862954662#code
 - V1 `BytecodeStorageReader` (arbitrum): https://arbiscan.io/address/0xa07f47c30C262adcC263A4D44595972c50e04db7#code
 
-## Minter Suite
```
Comment (ryley-o) [inline packages/contracts/README.md:156]: moved Minter Suite section up above # Deployments

Comment (jakerockland) [reply packages/contracts/README.md:156]: 🙏

File: packages/contracts/V3_ARCHITECTURE.md [hunk 2]
```diff
@@ -34,15 +34,20 @@ A diagram of the V3 core contract and Admin ACL contract integration is shown be
 
 The Randomizer contract is used to generate pseudo random numbers for the V3 core contract, when new tokens are minted. This architecture is designed to be highly flexible, enabling designs that may desire to use asynchrounous random number generation or other hash generation methods (such as our Polyptych Randomizer contract).
 
+A shared randomizer was developed as part of our shared minter suite development, and may be used to generate random numbers for all V3 tokens. The randomizer is designed to handle all globally approved shared minters, including minters that may assign token hashes directly such as the Polyptych minter.
```
Comment (jakerockland) [inline packages/contracts/V3_ARCHITECTURE.md:37]:
```suggestion
A shared randomizer was developed as part of our shared minter suite development, and may be used to generate pseudorandom numbers for all V3 tokens. The randomizer is designed to handle all globally approved shared minters, including minters that may assign token hashes directly such as the Polyptych minter.
```

File: packages/contracts/images/minter-suite-v2.png
Comment (jakerockland) [inline packages/contracts/images/minter-suite-v2.png:1]: I think we probably want to update this diagram to also reflect that core contract admins can allowlist on the minter filter one-off minters.

**Review by jakerockland — APPROVED** [review]
LGTM ser 🙏

File: packages/contracts/images/minter-suite-v2.png
Comment (ryley-o) [reply packages/contracts/images/minter-suite-v2.png:1]: good call - there are definitely some partners who will want to do that!
Updated - @jakerockland does the update look good to you?

**Comment by ryley-o** [comment]
heads-up @jakerockland - I've updated this in [b06ccf5](https://github.com/ArtBlocks/artblocks-contracts/pull/914/commits/b06ccf5e046ffc5223756fef05858dbcc6ce7e75) to ensure we also support legacy minter suite documentation while we migrate to the new minter suite. Those changes were very straightforward, and mostly un-deleting some of the portions previously removed in this PR.
Based on that, I'm going to go ahead and still consider the approval valid, and if there are any retroactive pick-ups you would like, let me know and we can implement in a follow-on PR.

**Pull request merged**